Dockerfile

```diff
@@ -0,0 +1,27 @@
+FROM rust:1.82.0-bullseye as base
```

Comment:
During Docker build, I get the following Dockerfile lint error (relatively new feature):
```
WARN: FromAsCasing: 'as' and 'FROM' keywords' casing do not match (line 1)
```

Also, this should probably have a `# syntax` tag:
```
# syntax=docker/dockerfile:1
```
See https://docs.docker.com/reference/dockerfile/#syntax